tests/test_text_generation_example.py

```diff
@@ -137,8 +137,11 @@ def _test_text_generation(
     if "llama" in model_name.lower():
         command += ["--trim_logits", "--attn_softmax_bf16"]
 
-    if "falcon" in model_name.lower():
+    if "falcon, starcoder2"in model_name.lower():
```

Comment:
This won't work..sent you a patch fix and also `make style` changes

Reply:
Thanks added.